src/app/tasks/task-ilo-alignment/modals/task-ilo-alignment-modal/task-ilo-alignment-modal.coffee

```diff
@@ -52,6 +52,7 @@ angular.module('doubtfire.tasks.task-ilo-alignment.modals.task-ilo-alignment-mod
         if response.data.error?
           alertService.add("danger", "Error: " + response.data.error, 6000)
     )
+    $scope.closeModal()
```

Comment:
If I don't call this function here, the rating component will call the alignment object still and it will raise a null object warning.
I will find a better solution here.